modules/storage-persistent-storage-reclaim-manual.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+
+[id="reclaim-manual_{context}"]
```

Comment:
Should this `id` include "pv"? For example, `reclaim-pv-manual`.

Reply:
I don't think so, it's just verifying what's stated in the intro...namely, that the previous claimant's data is still there.

Reply:
I think I was basing it off filename syntax in the directory, so I'll leave as is.